workflows/cloudify_system_workflows/snapshots/snapshot_restore.py

```diff
@@ -152,6 +154,7 @@ def restore(self):
             self._trigger_post_restore_commands()
             ctx.logger.debug('Removing temp dir: {0}'.format(self._tempdir))
             shutil.rmtree(self._tempdir)
+            os.remove(SNAPSHOT_RESTORE_MARKER_FILE_PATH)
```

Comment:
maybe encapsulate the add and the removing to a single location so there will be two func for on and off